algorithms/kernel/svm/svm_train_common_avx512_impl.i

```diff
@@ -68,36 +67,36 @@ void SVMTrainTask<float, daal::algorithms::svm::interface2::Parameter, avx512>::
     __m512 tau_vec    = _mm512_set1_ps(tau); // vector of tau's
 
     // mask_1: condition _I[j] is in I_low
-    // mask_2: condition -ygrad = y[i]*grad[i] >= (GE) -GMax.
+    // mask_2: condition grad = grad[i] < (GE) GMin.
     // mask_3: condition a > tau
 
     size_t j_cur;
     for (j_cur = jStart; (j_cur + 16) <= jEnd; j_cur += 16)
     {
         // if (jEnd > jCur)  make mask?
         DAAL_ASSERT(j_cur <= services::internal::MaxVal<int>::get())
-        __m512i Bj_vec_cur = _mm512_set1_epi32((int)(j_cur));
-        __m128i vec_I      = _mm_load_si128((__m128i *)(&_I[j_cur]));
-
-        // vector multiplication y_grad[jcur:jcur+16] = _y[jcur:jcur+16] * _grad[jcur:jcur+16]
-        __m512 valYGrad = _mm512_mul_ps(_mm512_loadu_ps(&_y[j_cur]), _mm512_loadu_ps(&_grad[j_cur]));
+        __m128i vec_I = _mm_loadu_si128((__m128i *)(&I[j_cur]));
 
         // if ((_I[j] & low) != low) { continue; }
         __mmask16 mask1_ = _mm_cmpeq_epi8_mask(_mm_and_si128(vec_I, vecAllLow), vecAllLow);
 
-        // find max(y[j]*grad[j]) instead - here's where we find vector of max's.
+        __m512i Bj_vec_cur = _mm512_set1_epi32((int)(j_cur));
+        // vector grad[jcur:jcur+16] = _grad[jcur:jcur+16]
+        __m512 valGrad = _mm512_loadu_ps(&grad[j_cur]);
```

Comment:
seems it's better to use `_mm512_load_ps` where it is possible (for internally allocated memory we can guaranty alignment)